packages/agent-toolkit/src/core/tools/monday-apps-tools/consts/routes.consts.ts

```diff
@@ -6,47 +6,34 @@ export enum HttpMethod {
 }
 
 export const MONDAY_APPS_DOMAIN = 'https://monday-apps-ms.monday.com';
-const APPS_MS_PATH = '/apps_ms';
 
 const BASE_API = '/api';
 const BASE_APPS = `${BASE_API}/apps`;
 const BASE_APP_VERSIONS = `${BASE_API}/app-versions`;
 const BASE_CODE = `${BASE_API}/code`;
 const BASE_STORAGE = `${BASE_API}/storage`;
 
-const API_URL = `${MONDAY_APPS_DOMAIN}${BASE_API}`;
 const APPS_URL = `${MONDAY_APPS_DOMAIN}${BASE_APPS}`;
 const APP_VERSIONS_URL = `${MONDAY_APPS_DOMAIN}${BASE_APP_VERSIONS}`;
 const CODE_URL = `${MONDAY_APPS_DOMAIN}${BASE_CODE}`;
 const STORAGE_URL = `${MONDAY_APPS_DOMAIN}${BASE_STORAGE}`;
-const APPS_MS_URL = `${MONDAY_APPS_DOMAIN}${APPS_MS_PATH}`;
 
 export const API_ENDPOINTS = {
   APPS: {
     GET_ALL: APPS_URL,
     CREATE: APPS_URL,
     CREATE_FROM_MANIFEST: `${APPS_URL}/manifest`,
-    GET_MANIFEST: (appId: number) => `${APPS_URL}/${appId}/manifest`,
```

Comment:
Why not? Maybe the agent can use it to have example of a manifest 
Or this might have some PII

Reply:
it's not used anywhere